lib/cloudinary.rb

```diff
@@ -44,10 +44,9 @@ def self.user_platform
 
   def self.USER_AGENT
     if @@user_platform.empty?
-      "#{USER_AGENT}"
-    else
-      "#{@@user_platform} #{USER_AGENT}"
+      USER_AGENT
```

Comment:
add `return` statement here, or revert `else` clause, otherwise the following line is always returned